goa/proj.android-studio/goa/build.gradle

```diff
@@ -27,4 +27,24 @@ dependencies {
     implementation 'com.android.support.constraint:constraint-layout:1.1.3'
     implementation project(':libcocos2dx')
     compile project(path: ':downloader_library')
+    implementation fileTree(include: ['*.jar'], dir: 'libs')
```

Comment:
Review usage of these libraries. 
Just check if you can use the specific jar file name instead of *